lib/NormalModuleFactory.js

```diff
@@ -121,7 +122,7 @@ function NormalModuleFactory(context, resolvers, options) {
 				if(resource === false)
 					return callback(null,
 						new RawModule("/* (ignored) */",
-							"ignored " + context + " " + request,
+							      "ignored " + path.relative(__dirname, context) + " " + request,
```

Comment:
I wonder if `"ignored"` is enough here:

`new RawModule("/* (ignored) */", "ignored", "ignored")`